src/com/facebook/buck/cli/PublishCommand.java

```diff
@@ -226,6 +234,40 @@ public BuildTargetSpec apply(TargetNodeSpec input) {
           .build();
     }
 
+    if (includeJavadoc) {
+      specs = ImmutableList.<TargetNodeSpec>builder()
+          .addAll(specs)
+          .addAll(FluentIterable
+              .from(specs)
+              .filter(
+                  new Predicate<TargetNodeSpec>() {
+                    @Override
+                    public boolean apply(TargetNodeSpec input) {
+                      if (!(input instanceof BuildTargetSpec)) {
+                        throw new IllegalArgumentException(
+                            "Targets must be explicitly defined when using " +
```

Comment:
The other buck commands all allow path specs to be used, but for some reason publishing is different. *sigh* More of an informational comment than anything else.